app/main/views/jobs.py

```diff
@@ -191,17 +190,11 @@ def view_notifications(service_id, message_type=None):
 @main.route("/services/<uuid:service_id>/notifications.json", methods=["GET", "POST"])
 @main.route("/services/<uuid:service_id>/notifications/<template_type:message_type>.json", methods=["GET", "POST"])
 @user_has_permissions()
-def get_notifications_as_json(service_id, message_type=None):
-    return jsonify(get_notifications(service_id, message_type, status_override=request.args.get("status")))
+def _get_notifications_dashboard_partials_data_as_json(service_id, message_type=None):
```

Comment:
Not sure I love the naming here:
- I think of the dashboard as the page with the three blue boxes. I’d call this page the notifications page
- I can’t find other examples where we use underscore prefixing to mean an function which serves partials

Given those points I’d go for something more like:
```suggestion
def get_notifications_page_partials_as_json(service_id, message_type=None):
```

Reply:
Nice, will rename. I suppose I was using the underscore prefix to indicate that it wasn't a view intended for public consumption (ie semi internal) moreso than one for partials.